templates/templates/_tag_manager.html

```diff
@@ -10,11 +10,33 @@
         }
     </script>
     <!-- Google Tag Manager -->
-    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-    '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','GTM-K92JCQ');</script>
-    <!-- End Google Tag Manager -->
+     <script>
+        document.addEventListener('DOMContentLoaded', () => {
+          /** init gtm after 3 seconds - can be adjusted */
+          setTimeout(initGTM, 3000);
+        });
+        document.addEventListener('scroll', initGTMOnEvent);
+        document.addEventListener('mousemove', initGTMOnEvent);
+        document.addEventListener('touchstart', initGTMOnEvent);
```

Comment:
```suggestion
        document.addEventListener('touchstart', initGTMOnEvent);
        document.addEventListener('keydown', initGTMOnEvent);
```
For users navigating with keyboard